lambdas/service/app.py

```diff
@@ -645,46 +643,90 @@ def generate_manifest(event, context):
     return {'Location': response.headers['Location']}
 
 
-@app.route('/dss/files/{uuid}', methods=['GET'], cors=True)
+proxy_endpoint_path = "/fetch/dss/files"
+
+
+@app.route(proxy_endpoint_path + '/{uuid}', methods=['GET'], cors=True)
 def files_proxy(uuid):
     """
-    Proxies requests to DSS' /files endpoint, with the intend of implementing Retry-After on the client's behalf. The
-    file URL that this service returns on the /repository/files endpoint points at this endpoint. This proxy endpoint
-    was added to prevent browsers that don't implement Retry-After from quickly exhausting their limit on the number
-    of retries.
+    Initiate checking out file for download from the data store
```

Comment:
https://github.com/DataBiosphere/azul/pull/571#discussion_r239572882

I'm getting really frustrated with the apparent inability to just copy and paste my suggestions. This is not the first time that I gave a suggested change verbatim and it was botched even then.  You can also just click "Commit suggestion".